go.mod

```diff
@@ -1,6 +1,8 @@
 module github.com/rook/rook
 
-go 1.20
+go 1.21
```

Comment:
We can't update the go version in release branches, we need to make the package updates without this version change